doc/man3/EVP_PKEY_CTX_set1_scrypt_salt.pod

```diff
@@ -0,0 +1,158 @@
+=pod
```

Comment:
I'd suggest the documentation follows the style of other algorithms like RSA-PSS: that is located in man7 and named after the algorithm itself. There are some existing KDFs (like the TLS1 PRF) that don't yet follow this but they will be updated at some point.